platform/mbed_stats.h

```diff
@@ -138,14 +140,14 @@ typedef enum {
  * struct mbed_stats_sys_t definition
  */
 typedef struct {
-    uint32_t os_version;                /**< Mbed OS Version (Release only) */
-    uint32_t cpu_id;                    /**< CPUID Register data (Cortex-M only supported) */
-    mbed_compiler_id_t compiler_id;     /**< Compiler ID \ref mbed_compiler_id_t */
+    uint32_t os_version;                /**< Mbed OS version (populated only for tagged releases) */
+    uint32_t cpu_id;                    /**< CPUID register data (Cortex-M only supported) */
+    mbed_compiler_id_t compiler_id;     /**< Compiler id \ref mbed_compiler_id_t */
     uint32_t compiler_version;          /**< Compiler version */
 } mbed_stats_sys_t;
 
 /**
- *  Fill the passed in sys stat structure with system stats.
+ *  Fill the passed in system stat structure with system statistics.
```

Comment:
Query: Should we spell out the first "stat" in the this sentence match the others, or does it stand for something other than statistics?

Reply:
lets have it statistics everywhere